tests/atomistics/structure/test_analyse.py

```diff
@@ -0,0 +1,28 @@
+# coding: utf-8
+# Copyright (c) Max-Planck-Institut für Eisenforschung GmbH - Computational Materials Design (CM) Department
+# Distributed under the terms of "New BSD License", see the LICENSE file.
+
+import unittest
+import numpy as np
+from pyiron.atomistics.structure.atoms import CrystalStructure
+
+
+class TestAtoms(unittest.TestCase):
+    @classmethod
```

Comment:
Nit: If these are just passed I'm pretty sure they can be omitted without the code complaining.